app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -98,125 +75,35 @@ function ReviewIssuesStep({
     changeStepCanComplete(!!hasDismissed);
   }, [hasDismissed]);
 
+  if (!hasDismissed && pii) {
+    return <BarcodeAttentionWarning onDismiss={onWarningPageDismissed} pii={pii} />;
+  }
+  // Show warning screen
   if (!hasDismissed) {
```

Comment:
This is implicitly `if (!hasDismissed && !pii) {` because of the statement above. I'd probably change the code to `if (!hasDismissed && !pii) {` so the intent is clearer.